src/net/ifaddrs.c

```diff
@@ -6,6 +6,7 @@
 #include <unistd.h>
 #include <sys/socket.h>
 #define __USE_MISC 1   /**< Use MISC code */
+#define _BSD_SOURCE
```

Comment:
is it possible to use `_DEFAULT_SOURCE` instead ?

I think we are using that one in most places now

Reply:
Unfortunately, musl uses 
`#if defined(_GNU_SOURCE) || defined(_BSD_SOURCE)`
to guard definitions of IFF_UP et al, so this is not possible.